pkgs/tools/networking/mosh/default.nix

```diff
@@ -4,11 +4,11 @@
 
 stdenv.mkDerivation rec {
   pname = "mosh";
-  version = "1.3.2";
+  version = "1.4.0";
 
   src = fetchurl {
```

Comment:
No I just went with the principle of minimum changes to submit the patch. 😁
In general, on nixpkgs, is `fetchFromGitHub` preferred? If so, why? I have no opinions on this, I'm just interested in what's considered the golden standard.

Reply:
Yeah, `fetchFromGitHub` is preferred because it is smart on if it downloads from the tarball or a git clone depending on the inputs. Also, it looks more cleaner and it is easier to review (the owner/repo names are attributes instead of being hidden in the URL).

Reply:
I think if we have those special fetchers for some platforms, it's only consistent to use them. Also, `fetchFromGithub` knows best where github stores it's releases in what format and how desirable those formats are, but don't ask me details :wink:

Reply:
There is also fetchFromGitea, fetchFromGitLab, FetchFromSourceHut and they are always perfered.